omnibus-arbitrum-sepolia.toml

```diff
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "10"
+version = "11"
```

Comment:
Not removing from sepolia because already deployed. Just flipping the switch to disable deposits.